App/xcbuddykit/Sources/Commands/InitCommand.swift

```diff
@@ -67,6 +81,12 @@ public class InitCommand: NSObject, Command {
                                      kind: Bool.self,
                                      usage: "Generate xcodeproj after create Project.swift file",
                                      completion: .none)
+        
+        interactiveArgument = subParser.add(option: "--interactive",
+                                         shortName: "-i",
+                                         kind: Bool.self,
+                                         usage: "Launch wizard to request project details",
```

Comment:
I think people are more familiar with `scaffolding` to name the process of creating a new project following a few steps. Yeoman, which is a tool for that purpose in web projects http://yeoman.io uses:

> THE WEB'S SCAFFOLDING TOOL FOR MODERN WEBAPPS

What do you think?

Reply:
I didn't know this word, but I think make more sense than use `interactive` name. thanks :)